docs/swagger/spec.yml

```diff
@@ -591,6 +591,41 @@ paths:
           description: The UUID parameter does not refer to a project or the user does not have access to the project it refers to
           schema:
             $ref: '#/definitions/Error'
+
+  /projects/{uuid}/aoi/:
+    get:
+      summary: Get all Areas of Interest for this project.
+      parameters:
+        - $ref: '#/parameters/uuid'
+      responses:
+        200:
+          description: Json summary of the filters and Polygons that make up AOIs for the given project.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/AOI'
```

Comment:
By default every list endpoint should be paginated (and is I think) and not paginating would be unexpected.

Who knows how many AOIs people will produce for a project, unless we put an upper bound on it.

We have regretted not paginating from the start on other products so we do it for every list endpoint by default.